src/audio.cpp

```diff
@@ -0,0 +1,31 @@
+#include "audio.hpp"
+#include "config.h"
+
+#ifdef HAVE_PULSE
+#include "pulse.hpp"
+#endif
+
+#ifdef HAVE_PIPEWIRE
+#include "pipewire.hpp"
+#endif
+
+AudioReader *AudioReader::create(AudioReaderParams params)
+{
+#ifdef HAVE_PIPEWIRE
```

Comment:
This got me thinking, if we eventually want/need to support even more audio backends (like ALSA), maybe we should add a new command line argument for selecting the audio capture backend? It could be a long option only.